int32 x
int32 y

int32 SIZE=3
int32[3] sized_demo

int32[] no_size_demo